frontend/src/components/chat/chat-list.tsx

```
@@ -10,84 +11,81 @@ import Markdown from 'react-markdown';
 import remarkGfm from 'remark-gfm';
 import { INITIAL_QUESTIONS } from '@/utils/initial-questions';
 import { Button } from '../ui/button';
+import { Message } from '../types';
+import { useAuth } from '@/app/hooks/useAuth';
+
+// Helper function to determine if a message is from the user
+const isUserMessage = (role: string) =>
+  role.toLowerCase() === 'user' || role.toLowerCase() === 'User';
```

Biome

[error] This expression always returns false, because the string is converted and will never match

This call convert the string to lower case

... but this value is not in lower case

Unsafe fix: Use lower case string value.


(lint/correctness/noStringCaseMismatch)